src/Microsoft.AspNetCore.Hosting/Internal/HostingApplication.cs

```diff
@@ -9,11 +9,17 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
 
 namespace Microsoft.AspNetCore.Hosting.Internal
 {
     public class HostingApplication : IHttpApplication<HostingApplication.Context>
     {
+#if !NET451
+        private const string ActivityName = "Microsoft.AspNetCore.Hosting.Activity";
```

Comment:
Sigh, I see that we named the outgoing Http request System.Net.Http.Activity (I did not catch that in review), a much better name for that is System.Net.Http.HttpReqOut.   It may not be too late to fix that.  I will research this.